spark-etl/src/main/scala/geotrellis/spark/etl/config/DefaultEtlConf.scala

```diff
@@ -0,0 +1,87 @@
+package geotrellis.spark.etl.config
+
+import geotrellis.spark.etl.config.json._
+
+import org.apache.spark.SparkContext
+import com.github.fge.jackson.JsonLoader
+import spray.json._
+
+trait DefaultEtlConf extends ConfigParse {
```

Comment:
The name default is confusing. Should it be BaseEtlConfig? Perhaps ask the room, if I'm the only one who thinks so then perhaps we keep it as is